src/app/snapshot/components/vm-snapshot-list-view/vm-snapshot-list-view.component.ts

```diff
@@ -1,16 +1,27 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
```

Comment:
this looks identical to `vm-snapshot-card-view.component.ts`. Maybe we can fix this?